Assync

SergeySamodurovS opened the pull request:
minor changes #9

A1tehr committed:
Change void on Task

SergeySamodurovS committed:
Update DBconnection.cs

Change type CloseAsync() frov void to Task

anst-foto commented on DBconnect/DBconnection.cs:
```diff
@@ -258,7 +258,7 @@ public void Close()
                 error?.Invoke("Соединенине с БД либо уже закрыто, либо его не было");
```
```suggestion
                Error?.Invoke("Соединенине с БД либо уже закрыто, либо его не было");
```
Другое соглашение об именовании переменных в этом проекте было? Если нет, то имена эвентов и делегатов пишутся с большой буквы

anst-foto reviewed (changes requested):
Привести имена переменных, эвентов, экземпляров классов к нашему соглашению об именовании

SergeySamodurovS committed:
Update DBconnection.cs

Приведение имён делегатов и переменных к требуемым Соглашением об именовании переменных

anst-foto reviewed (approved):
Одобряю
